cmd/fs-v1-multipart.go

```diff
@@ -361,9 +361,9 @@ func (fs fsObjects) PutObjectPart(bucket, object, uploadID string, partID int, s
 		return "", toObjectErr(err, minioMetaMultipartBucket, uploadIDPath)
 	}
 
+	// Append the part in background.
+	errCh := fs.bgAppend.append(fs.storage, bucket, object, uploadID, fsMeta)
```

Comment:
But we do `errCh <- errAppendPartsTimeout` on the other side so receive on errCh would not block.